requirements.txt

```diff
@@ -0,0 +1,8 @@
+-e git+http://github.com/openaustralia/scraperwiki-python.git@morph_defaults#egg=scraperwiki
```

Comment:
[ScraperWiki classic](https://classic.scraperwiki.com/) was retired a few years ago. They still host scrapers on their new platform (with some nice features like hourly runs) but the datasets aren't public and the feature itself is pretty buried in the interface.